download.py

```diff
@@ -0,0 +1,22 @@
+from piazza_api import Piazza
+import pprint
+import json
+
+p = Piazza()
+p.user_login()
+
+eecs183 = p.network("iiw41ofw7igeu")

```

Comment:
This name is outdated. Maybe name it something generic and pass the class code on the command line?